### PR TITLE
small ui fixes

### PR DESCRIPTION
**Pull request by kevodwyer**
- signup password safety tickbox must be ticked when creating account
- cancel spinner after navigating to root directory

### CODE REVIEW

File: src/components/filesystem/index.js [hunk 1]
```diff
@@ -426,6 +426,9 @@ module.exports = {
 
         goHome: function() {
             this.changePath("/");
+            Vue.nextTick(function() {
```
Comment (ianopolous) [inline src/components/filesystem/index.js]: Why isn't this being done the same as any other path change?

Comment (kevodwyer) [reply src/components/filesystem/index.js]: On subsequent 'home' button clicks I can see that vue.js is creating a spinner after the directory is re-calculated. Put breakpoint on existing line console.debug('Spinner module created!'). I don't know why. Adding the Vue.nextTick line ensures that the spinner is hidden again.

Comment (ianopolous) [reply src/components/filesystem/index.js]: It doesn't make sense to me why this would only affect /.

Comment (kevodwyer) [reply src/components/filesystem/index.js]: fixed a different way now

Comment (ianopolous) [reply src/components/filesystem/index.js]: Nice!

**Pull request merged**